browser/css/jssidebar.css

```diff
@@ -10,6 +10,12 @@
 	vertical-align: middle;
 }
 
+.ui-expander-content.sidebar.expanded > .sidebar .ui-grid-cell {
```

Comment:
won't be that rule needed not only in expander in sidebar but also in any other grid-cell ?

Reply:
and in all jsdialogs not only in sidebar?

Reply:
as we are editing jssidebar.css from my point of view there should be ONLY sidebar related css rules. If we want them to have also for jsdialogs, than I would move them to jsdialogs. Which would mean more testing, cause it has to work also at dialgoues.